components/PrettyDuration/index.jsx

```diff
@@ -0,0 +1,16 @@
+export default function PrettyDuration(seconds) {
```

Comment:
questo non è un componente react ma una funzione quindi non dovrebbe iniziare con la maiuscola, ne stare sotto 'components/' ma 'utils/'.

Magari la chiamerei in modo meno vago "formatVideoDuration"?